### ISSUE TITLE
repo owner

### ISSUE THREAD

**Issue by patzm** [aleju/imgaug#764]
@aleju, it appears that you stopped being active on GitHub around June last year. I personally hope that everything is fine!

I open this issue to see if there are collaborators that actually have full access to this repo and can do releases. If not, the main question would be if you, @aleju, would be open to someone joining the effort of carrying forward this awesome library. I think many people out there really like and depend on it.

**Comment by half-potato** [comment]
A little bit of stalking reveals that he is still alive and working at Tensoreye.

**Comment by patzm** [comment]
lol, I live about 1km away from them. I could also walk over and ask him :laughing:

**Comment by HamsterHuey** [comment]
Yes, this would really be great. I can certainly imagine the Herculean effort it has taken this library to get where it is today and it is totally understandable if this repo won't be able to benefit from as much focus from @aleju , but it would be a real shame for this repo to end up decaying with no transition plan to keep it alive and well as there are a lot of folks who rely on it at this point. It would be great if one or more developers could be pulled in to help with maintaining and maybe further development. If not, perhaps if there were some sort of officially blessed fork, that might also be okay.

It would just be a shame to see such a great library end up frozen in time at the current commit with no future bug fixes and updates.

**Comment by patzm** [comment]
Maybe @aleju could use https://adoptoposs.org/ to find co-owners :thinking:.

**Comment by SchernHe** [comment]
Any news?

**Comment by mauvilsa** [comment]
@patzm did you go in person to ask @aleju? There hasn't been any response to this in a long time. Is it time to think about an official community maintained fork of this project?

**Comment by patzm** [comment]
No I didn't. Sounds like a good idea though. Quite sad how this got dropped.

**Comment by gui-miotto** [comment]
It is really sad that this project was abandoned like that. I've been using it for years, but because of its current situation, I'm now inclined to migrate to [albumentations](https://albumentations.ai/).

I wonder if anyone has experience with both packages and could give a first-person account on how they compare. Google was not so helpful here.

**Comment by jspaezp** [comment]
@gui-miotto I think this one was much better documented for starters, and included visualization apis.
Albumentations does seem to be faster in my problems.
